pkg/network/common/egressip.go

```diff
@@ -37,7 +38,7 @@ type namespaceEgress struct {
 	vnid         uint32
 	requestedIPs []string
 
-	activeEgressIP string
+	activeEgressIP map[string]string
```

Comment:
so you use maps for this everywhere, but no one ever uses it actually as a _map_... it's more like an array of node-ip/egress-ip pairs. It might be better to make that explicit, and have

    type EgressIPAssignment {
            NodeIP   string
            EgressIP string
    }
    ...
    activeEgressIPs []EgressIPAssignment

(in particular, when it's a `map[string]string` it's really not obvious whether it's egressIP→nodeIP or nodeIP→egressIP)

Reply:
done